src/python/WMCore/WMBS/File.py

```diff
@@ -615,6 +615,8 @@ def addFilesToWMBSInBulk(filesetId, workflowName, files, isDBS = True,
                                 conn = conn,
                                 transaction = transaction)
     if len(runLumiBinds) > 0:
+        logging.info("AMR about to add run/lumi for %i files for fileset %i",
+                     len(runLumiBinds), filesetId)
```

Comment:
what is the structure of runLumiBinds? can we can the all the information we need?

Reply:
it's a list of dictionaries, with 2 keys/values, lfn and Run object.
Ideally we should print everything, the problem is that there are files (like MCFake) with > 100k lumi sections.
Maybe we could print lfn, run and # of lumis inside each run. What do you think?

Reply:
So len(runLumiBinds) only prints number of lfns?

> Maybe we could print lfn, run and # of lumis inside each run. What do you think?

Yes I think that will give more information.